routes/quiz.js

```diff
@@ -0,0 +1,15 @@
+const router = require('express').Router({ mergeParams: true })
+
+const {
+  roles: { TUTOR }
+} = require('~/consts/auth')
+const quizController = require('~/controllers/quiz')
+const asyncWrapper = require('~/middlewares/asyncWrapper')
+const { authMiddleware, restrictTo } = require('~/middlewares/auth')
+
+router.use(authMiddleware)
+
+router.use(restrictTo(TUTOR))
```

Comment:
I think, it is better to locate this middleware near authMiddleware

Reply:
i don't think that it is needed, cause there will be other routes, not restricted above this middleware

Reply:
It will look better, to locate middlewares together and routes with http methods together